### PR TITLE
Blog: clarify release notes for v14.21.2 and v16.19.0

### PR DESCRIPTION
**Pull request by richardlau**
cc @ljharb

### CODE REVIEW

File: locale/en/blog/release/v14.21.2.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ author: Richard Lau
 
 #### OpenSSL 1.1.1s
 
-This update is a bugfix release and does not address any security
-vulnerabilities.
+This OpenSSL version does not address any security vulnerabilities.
```
Comment (ljharb) [inline locale/en/blog/release/v14.21.2.md:15]: This change might not be needed, since it’s a patch release

**Review by ljharb — APPROVED** [review]
Thanks! It’d be ideal to also add “(SEMVER-MINOR)” to the semver-minor item(s) in the 16.19 commit list as well, since that’s how i usually scan to see what’s added :-)

**Comment by richardlau** [comment]
> Thanks! It’d be ideal to also add “(SEMVER-MINOR)” to the semver-minor item(s) in the 16.19 commit list as well, since that’s how i usually scan to see what’s added :-)

Updated.

**Review by ljharb — APPROVED** [review]
thanks!

**Review by Trott — APPROVED** [review]

**Pull request merged**